Makefile

```diff
@@ -37,7 +37,7 @@ test: check-env lint security
 				/opt/venv/bin/coverage report --fail-under=30"
 
 start: build
-		@docker compose -f docker-compose.yml -f docker-compose-${MODE}.yml up -d
+		@docker compose -f docker-compose.yml -f docker-compose-${MODE}.yml up --wait -d
```

Comment:
I believe this was already the behavior given the healthchecks defined in the docker-compose files. Was that not the case on your system?

Reply:
We are deploying AICA using Ansible, this is relevant because we always create a new system using the same method. Without the --wait it timeouts and always fails on first Ansible deploy. With the --wait we get a successful build